src/lib/validation/recipient.ts

```
@@ -0,0 +1,56 @@
+}
+
+export async function validateRecipient(recipient: string, type: RecipientType): Promise<string> {
+    switch (type) {
+        case 'ENS':
+            // check if the recipient is a valid ENS name
+            if (!recipient.endsWith('.eth')) {
+                throw new RecipientValidationError('Invalid ENS name format')
+            }
+
+            // resolve the ENS name to address
+            const resolved = await resolveFromEnsName(recipient)
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)